app/lib/features/bug_report/controllers/bug_report_controller.dart

```diff
@@ -1,8 +1,37 @@
+import 'dart:convert';
 import 'dart:io';
 
 import 'package:effektio_flutter_sdk/effektio_flutter_sdk.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
+import 'package:http/http.dart' as http;
+import 'package:http_parser/http_parser.dart';
+import 'package:path/path.dart';
+
+const rageshakeUrl = String.fromEnvironment(
+  'RAGESHAKE_URL',
+  defaultValue: 'http://localhost/api/submit',
+);
+
+const rageshakeUsername = String.fromEnvironment(
+  'RAGESHAKE_USERNAME',
+  defaultValue: 'alice',
+);
+
+const rageshakePassword = String.fromEnvironment(
+  'RAGESHAKE_PASSWORD',
+  defaultValue: 'secret',
+);
+
+const issueAppName = String.fromEnvironment(
+  'ISSUE_APP_NAME',
+  defaultValue: 'a3-nightly',
+);
+
+const issueAppVersion = String.fromEnvironment(
+  'ISSUE_APP_VERSION',
+  defaultValue: '1.0.0',
+);
```

Comment:
Let's prefix them all with the same `RAGESHAKE_` please ...

```suggestion
const issueAppName = String.fromEnvironment(
  'RAGESHAKE_APP_NAME',
  defaultValue: 'a3-nightly',
);

const issueAppVersion = String.fromEnvironment(
  'RAGESHAKE_VERSION',
  defaultValue: '1.0.0',
);
```

Reply:
Which one is correct?
![image](https://user-images.githubusercontent.com/48786056/223090098-153a1cb4-736b-4921-b94e-336b3734613f.png)

Reply:
`a3-nightly`

I was just referring to the name of the env-var.

Reply:
Appended `RAGESHAKE_` prefix.